kitsune/questions/views.py

```diff
@@ -630,7 +632,10 @@ def aaq(
             statsd.incr("questions.aaq.details-form")
         else:
             form = None
-            if search:
+            if product_config:
```

Comment:
is there any chance that we won't have a product_config? 

This is also referenced in the if statement above so maybe you could use a variable here for both places instead of `get`ting the product in both places.